src/encoded/static/scss/encoded/modules/_matrix.scss

```diff
@@ -3,6 +3,9 @@ $col-category-header-height: 200px;
 $row-category-cell-height: 22px;
 $row-data-header-width: 200px;
 
+$Homo-sapiens: #63326E;
```

Comment:
why is the variable name capitalized? Do you something more descriptive like "human-body-color:#63326E"?

Reply:
The capitalization is because the class name is generated from organismTerms. I could add toLowerCase() or append `-color` on all the class names but I'd rather keep them more or less as permutations of the organism terms names to avoid making things too much more complicated.

Your comment reminded me that these colors exist in both `_summary.scss` and `matrix.scss` and that I should definitely avoid this duplication so I moved all the hard-coded organism colors to `_base.scss`.